doc/sphinxext/numpy_ext/docscrape.py

```diff
@@ -13,7 +14,7 @@
 import pydoc
 from warnings import warn
 
-from nipype.external.six import StringIO
+from six import StringIO
```

Comment:
We could get rid of all dependencies from six. That'd be actually a good plan, with future and the standard_aliases is enough to make code compatible.